docs/sce_file_contents.md

```diff
@@ -0,0 +1,131 @@
+# Single-cell gene expression file contents
+
+Single cell or single-nuclei gene expression data (filtered or unfiltered) is provided for use with R as an RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html).
+This object contains the expression data, cell and gene metrics, associated metadata, and, in the case of multimodal data like CITE-seq, data from additional cell-based assays.
+
+We highly encourage you to familiarize yourself with the general object structure and functions available as part of the [`SingleCellExperiment` package](https://bioconductor.org/packages/3.13/bioc/html/SingleCellExperiment.html) from Bioconductor.
+Below we present some details about the specific contents of the objects we provide.
+
+To begin, you will need to load the `SingleCellExperiment` package and read the RDS file:
+
+```r
+library(SingleCellExperiment)
+sce <- readRDS("SCPCL000000_filtered.rds")
+```
+
+## Common fields
+
+### Expression counts
+
+For both unfiltered and filtered single-cell and single-nucleotide experiments, the primary cell by gene RNA-seq expression count data is provided as the `"counts"` assay.
```

Comment:
```suggestion
The `"counts"` assay of the `SingleCellExperiment` object for single-cell and single-nucleus experiments (both unfiltered and filtered) contains the primary RNA-seq expression data.
```
Something about this sentence wasn't sitting right with me (maybe partially because you accidentally said single-nucleotide 😆), but I also think you want to emphasize that the gene expression data is found in the `counts` assay, and less about the structure since the next few sentences explain that. So I reordered it and tried to get rid of some of the repetitiveness found in the next sentences.

Reply:
Oh you are right I just looked through some of the other docs and we are using single-nuclei. I think we should stick with that as I'm pretty sure that's also how 10X references it.